### PR TITLE
chore(provider): zabbix from mediatype to script

### PR DESCRIPTION
**Pull request by talboren**

### CODE REVIEW

File: docs/providers/overview.mdx [hunk 1]
```diff
@@ -24,7 +24,7 @@ By leveraging Keep Providers, users are able to deeply integrate Keep with the t
   title="Service Now"
   href="/providers/documentation/service-now-provider"
   icon={
-    <img src="https://img.logo.dev/service-now.com?token=pk_dfXfZBoKQMGDTIgqu7LvYg" />
+    <img src="https://img.logo.dev/servicenow.com?token=pk_dfXfZBoKQMGDTIgqu7LvYg" />
```
Comment (Matvey-Kuk) [inline docs/providers/overview.mdx:27]: <3

File: docs/overview/introduction.mdx [hunk 2]
```diff
@@ -1,26 +1,63 @@
 ---
 title: "Introduction"
-description: "Keep is an open-source alert management and automation tool that provides everything you need to collect, enrich and manage alerts effectively."
+description: "Keep is an open-source alert management and AIOps platform that is a swiss-knife for alerting, automation, and noise reduction."
```
Comment (Matvey-Kuk) [inline docs/overview/introduction.mdx:3]: <3

**Review by Matvey-Kuk — DISMISSED** [review]

**Review by shahargl — DISMISSED** [review]
lgtm

**Review by shahargl — APPROVED** [review]
lgtm

**Pull request merged**